src/io/bit_buf.rs

```diff
@@ -0,0 +1,170 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+//! Provides a way to ergonomically manipilate bytes of queues of bits.
```

Comment:
typo: manipilate

Also, do you mean "bytes as queues of bits" instead of "bytes of queues of bits"?